packages/styles/package.json

```diff
@@ -20,9 +20,9 @@
     "lint:fix": "stylelint 'src/**/*.scss' --fix",
     "lint": "stylelint 'src/**/*.scss'",
     "serve": "serve -p 5000",
-    "test:watch": "echo 'No tests available yet.'",
-    "test": "node src/test.js",
-    "test:styles": "npm run test"
```

Comment:
Please please please, be careful with prettifier, everything is broken now because of the non-allowed dangling comma in this package.json.